docs/content/docs/dev/table/sql/analyze.md

```diff
@@ -0,0 +1,364 @@
+
+# Partition table, collect row count and all columns statistics for partition1.
+table_env.execute_sql("ANALYZE TABLE Store PARTITION (date_sk='2022-8-8', id=1) COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and all columns statistics for partition1 and partition2.
+table_env.execute_sql("ANALYZE TABLE Store PARTITION (date_sk='2022-8-8') COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and all columns statistics for all partitions.
+table_env.execute_sql("ANALYZE TABLE Store COMPUTE STATISTICS FOR ALL COLUMNS");
+
+# Partition table, collect row count and specify column `amount`'s statistics for partition1.
+table_env.execute_sql("ANALYZE TABLE Store PARTITION (date_sk='2022-8-8', id=1) COMPUTE STATISTICS FOR COLUMNS(amount)");
+
+# Partition table, collect row count and specify column `amount` and `product`'s statistics for partition1 and partition2.
+table_env.execute_sql("ANALYZE TABLE Store PARTITION (date_sk='2022-8-8') COMPUTE STATISTICS FOR COLUMNS(amount, product)");
+
+# Partition table, collect row count and column `amount` and `product`'s statistics for all partitions.
+table_env.execute_sql("ANALYZE TABLE Store COMPUTE STATISTICS FOR COLUMNS(amount, product)");
+```
+{{< /tab >}}
+{{< tab "SQL CLI" >}}
+```sql
+Flink SQL> CREATE TABLE Orders (
+>  `user` BIGINT NOT NULl,
+>  product VARCHAR(32),
+>  amount INT,
+>  ts TIMESTAMP(3)
+> ) with (
+>  ...
+> );
+[INFO] Table has been created.
+
+Flink SQL> CREATE TABLE Store (
+>  `id` BIGINT NOT NULl,
+>  product VARCHAR(32),
+>  amount INT,
+>  `date_sk` BIGINT
+>  ) PARTITIONED BY (`date_sk`, `id`)
+>  ) with (
+>    ...
+>  );
+[INFO] Table has been created.
+
+Flink SQL> ANALYZE TABLE Orders COMPUTE STATISTICS;
+    
+Flink SQL> ANALYZE TABLE Orders COMPUTE STATISTICS FOR ALL COLUMNS;
+
+Flink SQL> ANALYZE TABLE Orders COMPUTE STATISTICS FOR COLUMNS(a);
+    
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8', id=1) COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8') COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Store COMPUTE STATISTICS;
+
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8', id=1) COMPUTE STATISTICS FOR ALL COLUMNS;
+
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8') COMPUTE STATISTICS FOR ALL COLUMNS;
+    
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8', id=1) COMPUTE STATISTICS FOR COLUMNS(amount);
+    
+Flink SQL> ANALYZE TABLE Store PARTITION (date_sk='2022-8-8') COMPUTE STATISTICS FOR COLUMNS(amount, product);
+    
+Flink SQL> ANALYZE TABLE Store COMPUTE STATISTICS FOR COLUMNS(amount, product);
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+
+## Syntax
+
+```sql
+ANALYZE TABLE [catalog_name.][db_name.]table_name [PARTITION(partcol1=val1 [, partcol2=val2, ...])] COMPUTE STATISTICS [FOR COLUMNS col1 [, col2, ...] | FOR ALL COLUMNS]
+```
+- PARTITION(partcol1=val1 [, partcol2=val2, ...]) is optional for the partition table
```

Comment:
PARTITION(partcol1=val1 [, partcol2=val2, ...]) is required for the partition table.

Reply:
> PARTITION(partcol1=val1 [, partcol2=val2, ...]) is required for the partition table.

done!